packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -639,6 +704,82 @@ const processTemplate = (relFile, inputDir, outputDir, context) => {
     }
 }
 
+/**
+ * Process the Application Extensions into the application-extensions directory.
+ *
+ * @param appExtensions - An array of the Application Extension names.
+ * @param extractAppExtensions - A boolean indicating whether to extract the Application Extensions code from the npm package
+ * @param appExtensionsDir - The path to the application-extensions directory.
+ */
+const processAppExtensions = (
+    appExtensions = [],
+    extractAppExtensions = false,
+    appExtensionsDir
+) => {
+    if (appExtensions.length > 0 && extractAppExtensions) {
+        appExtensions.forEach((appExtensionName) => {
+            const appExtensionTmp = fs.mkdtempSync(
+                p.resolve(os.tmpdir(), `extract-${appExtensionName}`)
+            )
+            const appExtensionTarFile = sh
+                .exec(
+                    `npm pack @salesforce/${appExtensionName} --pack-destination="${appExtensionTmp}"`,
```

Comment:
This kind of answers my own comment that I made previously do we want to have the values include the full package name? That might mean that when we incorporate third-party extensions that have different names spaces things will just work.